dsl/src/main/scala/org/http4s/dsl/RequestDsl.scala

```diff
@@ -37,6 +39,8 @@ trait RequestDsl extends Methods with Auth {
   val /: : impl./:.type
   val +& : impl.+&.type
 
+  def PathVar[A](cast: String => Try[A]): impl.PathVar[A]
+
```

Comment:
This is causing mima / bincompat issue, alternative would be not to include it on this PR or creating an extension trait.

Reply:
Opted to not include the interface addition/change